### DISCUSSION TITLE
Invalid CSS in "Custom color schemes" section

### DISCUSSION THREAD

**Discussion by galvana** [squidfunk/mkdocs-material#3130] [General]
This section uses the following example:

```
[data-md-color-scheme="youtube"] {
  --md-primary-fg-color:        #EE0F0F;
  --md-primary-fg-color--light: #ECB7B7;
  --md-primary-fg-color--dark:  #90030C;
}
```

As far as I could tell this is an issue for two reasons:
- Nesting like this is only available in SCSS
- The `extra_css` attribute in mkdocs.yml does not support .scss files

I'm curious to see what everyone's thoughts are on this since this might cause confusion for others in the future. Thanks!

**Comment by squidfunk** [comment]
This is valid CSS. It’s not a nested selector.

**Reply by galvana** [reply]
My mistake, the syntax issue was on my end. Thanks for the response!
